src/Resources/app/administration/src/module/findologic-module/page/findologic-page/findologic-page.html.twig

```diff
@@ -84,7 +84,8 @@
                                             :allConfigs="allConfigs"
                                             :actualConfigData="actualConfigData"
                                             :shopkeyErrorState="shopkeyErrorState"
-                                            :selectedSalesChannelId="selectedSalesChannelId">
+                                            :selectedSalesChannelId="selectedSalesChannelId"
+                                            :selectedLanguageId="selectedLanguageId">
```

Comment:
Why wasn't this required before?

Reply:
Why was this resolved? There was no code change and my question hasn't been answered.

Reply:
It was not required before because the implementation was done differently and it was checked directly in the module instead of the component.